langchain/src/vectorstores/closevector/common.ts

```diff
@@ -0,0 +1,227 @@
+import type { CloseVectorHNSWNode } from "closevector-node";
+import type { CloseVectorHNSWWeb } from "closevector-web";
```

Comment:
This will cause import issues unfortunately :(

You'll need to separate imports out into the individual entrypoints.

Reply:
@jacoblee93 

Ah! Sorry for not noticing that. I have deleted the import from closevector-node or closevector-web in common.ts; instead, I imported an type from a package named closevector-common, which is a package shared by closevector-node and closevector-web in latest code.